src/step-function.ts

```diff
@@ -504,6 +507,56 @@ abstract class BaseStepFunction<
 }`;
       },
     });
+
+    // Integration object for api gateway vtl
+    this.apiGWVtl = {
+      prepareRequest: (obj) => {
+        // TODO: this is currently broken. StepFunction interface requires a
+        // top level `input` key to be passed in but it shouldn't
```

Comment:
It requires that because you can also specify the step function name for idempotency

Reply:
We need to update the docs then: https://functionless.org/docs/concepts/step-function/usage#from-a-function-stepfunction-or-appsyncresolver

Reply:
I don't mind that although it's unorthodox.